src/graphics/gui.c

```diff
@@ -12,19 +15,64 @@
 int screen_width = INITIAL_SCREEN_WIDTH;
 int screen_height = INITIAL_SCREEN_HEIGHT;
 char title[] = GAME_TITLE;
-
 int state = STATE_START;
 
-void runGameGui(){
+void runGameGui() {
+
     MenusBundle *menu_bundle = initMenusBundle(&screen_width, &screen_height);
+    GameplayData *gameplay_data = NULL;
+
     InitWindow(screen_width, screen_height, title);
+    Vector2 mouse_position = {0.0f, 0.0f};
     SetTargetFPS(60);
-    Vector2 mousePos = {0.0f, 0.0f};
-    Vector2 *mp = &mousePos;
-    while (!WindowShouldClose() && state != STATE_EXIT){
-        handleStateClicks(menu_bundle, mp, &state);
-        handleGUIDraw(menu_bundle, mp, &state, &screen_width, &screen_height);
+
+    // TODO: Move to a different file.
```

Comment:
Yes 👍

Reply:
Handled by this JIRA ticket: https://suchicodes.atlassian.net/browse/MR-13